run-tests.sh

```diff
@@ -1,73 +1,21 @@
 #!/usr/bin/env bash
 
-
 set -eu
 
-declare status_code
+remove_quotes () {
+  echo "$1" | tr -d '"'
+}
+
+declare error_code
 # shellcheck disable=SC2154
-# status_code="$(curl --silent --output /dev/null --write-out '%{http_code}' "$CFN_HelloWorldApi")"
-status_code="$(curl --silent --output /dev/null --write-out '%{http_code}' "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200")"
+#The CFN variables seem to include quotes when used in tests these must be removed before assigning them.
+CFN_CICBackendURL_NoQuotes=$(remove_quotes $CFN_CICBackendURL)
+export DEV_CRI_CIC_API_URL=`echo ${CFN_CICBackendURL_NoQuotes%/}`
+export DEV_IPV_STUB_URL="https://erveje5km8.execute-api.eu-west-2.amazonaws.com/dev/start"
+
+cd ./src; npm run test:e2e
+error_code=$?
+
+cp -rf $TEST_REPORT_DIR $TEST_REPORT_ABSOLUTE_DIR
```

Comment:
```suggestion
cp -rf "$TEST_REPORT_DIR" "$TEST_REPORT_ABSOLUTE_DIR"
```
prevent unexpected before from word splitting if the env vars contain spaces